src/test/java/com/uber/cadence/workflow/ManualActivityCompletionWorkflowTest.java

```diff
@@ -0,0 +1,249 @@
+/*
+ *  Copyright 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Is this correct?

Reply:
oh, I didn't see the "modifiications copyright" down below, I guess it is from there. the licence URL looks sus, but I guess it is what it is.

Reply:
The cadence java client is a fork of https://github.com/aws/aws-swf-flow-library 😄